recipes/ckanweb-deploy-exts.rb

```diff
@@ -349,6 +350,16 @@
 			user "#{account_name}"
 			command "sed -i 's|^\\(use\s*=\\)\\(.*:FriendlyFormPlugin\\)|#\\1\\2\\n\\1 ckanext.csrf_filter.token_protected_friendlyform:TokenProtectedFriendlyFormPlugin|g' #{config_dir}/who.ini"
 		end
+
+		bash "Disable built-in CKAN CSRF filter" do
+			user "#{account_name}"
+			cwd "#{config_dir}"
+			code <<-EOS
+				if [ -z "$(grep 'WTF_CSRF_ENABLED' #{config_file})" ]; then
```

Comment:
do you have a way to swap this back to on?

Reply:
The config file is recreated from template every time we run 'configure', so this flag will disappear as soon as it's not being reapplied.